code/game/machinery/vending.dm

```diff
@@ -4107,3 +4107,61 @@ var/global/list/obj/item/weapon/paper/lotto_numbers/lotto_papers = list()
 	new hiddenmouse(get_turf(src))
 
 
+/obj/machinery/vending/art
+	name = "\improper Le Patron des Arts"  //Mécène won't render
```

Comment:
It won't? It should. I guess we have some more Unicode snooping to do.

Reply:
It renders on the machine but not in the browse() window

Reply:
Oh, right. It is possible to make it so that it would, but it's a pain in the ass without replacing the bare `browse()` with a browser datum, etc.